checks/tasks/__init__.py

```diff
@@ -38,10 +38,10 @@ def ub_ctx(self):
                 self._ub_ctx.zone_add("test.", "transparent")
 
             self._ub_ctx.set_option("cache-max-ttl:", str(settings.CACHE_TTL * 0.9))
+            self._ub_ctx.set_option("cache-max-negative-ttl:", str(settings.CACHE_TTL * 0.9))
             # Some (unknown) tests probably depend on consistent ordering in unbound responses
             # https://github.com/internetstandards/Internet.nl/pull/613#discussion_r892196819
             self._ub_ctx.set_option("rrset-roundrobin:", "no")
```

Comment:
Btw, the comment about rrset-roundrobin is for tests that get back a DNS answer but only care about the first X records from the results like the mailtest where only the first 8(?) mailservers are checked. Because this call to Unbound could be done from different places, turning off roundrobin guarantees that the same 8 in the mailtest for example will be used in different points of the code.